bedrock/mozorg/templates/mozorg/home/home-en.html

```diff
@@ -67,14 +67,15 @@ <h1>{{ self.page_title() }}</h1>
     ] -%}
     {% include promos|random() %}
   {% else %}
-    {% call hero(
-      title='Firefox products are designed to protect your privacy',
-      class='privacy-promise-hero mzp-has-image mzp-t-dark mzp-t-firefox',
+    {% call split(
       image_url='img/firefox/privacy/promise/privacy-hero.png',
       include_highres_image=True,
-      include_cta=True,
-      heading_level=2
+      media_after=True,
+      media_class='mzp-l-split-h-center',
+      block_class='mzp-t-split-nospace mzp-l-split-center-on-sm-md mzp-t-dark mzp-t-firefox',
+      theme_class=''
```

Comment:
```suggestion
      theme_class='mzp-t-dark mzp-t-firefox'
```

I wonder if you can scope the firefox brand variable to the mzp-t-firefox class on this mozilla page